diff --git a/python/pyproject.toml b/python/pyproject.toml
index 699c039e1c2f399c0fc0aab6976d9ba41dd0f7c1..57f2c58f1b97f9117d7a2a8386dc2d48b94980b5 100755
--- a/python/pyproject.toml
+++ b/python/pyproject.toml
@@ -83,7 +83,7 @@ dependencies = [
   "tiktoken",
   "tilelang==0.1.12",
   "timm==1.0.16",
-  "tokenizers==0.22.2", # 0.23.0rc0 is incompatible with transformers' CLIPTokenizer.
+  "tokenizers==0.23.1",
   "tokenspeed_mla==0.1.8",
   "tomli ; python_version < '3.11'",
   "torch==2.13.0",
@@ -92,7 +92,7 @@ dependencies = [
   "torchcodec==0.15.0 ; sys_platform != 'linux' or (sys_platform == 'linux' and platform_machine != 'aarch64' and platform_machine != 'arm64' and platform_machine != 'armv7l')", # Not available on Linux ARM.
   "torchvision",
   "tqdm",
-  "transformers==5.12.1",
+  "transformers==5.17.0",
   "uvicorn",
   "uvloop",
   "watchfiles",
